src/db/migrations/20190405011452-create-article-likes.js

```diff
@@ -1,42 +1,18 @@
 export default {
   up: (queryInterface, Sequelize) =>
     queryInterface.createTable('ArticleLikes', {
-      id: {
```

Comment:
why did we have to remove all these lines of code?

Reply:
It is so because I built the 'Likes' table as a pivot to have just two columns , user_id and article_id.

The id column, createdAt, and updatedAt do not matter to my implementation, so I removed them.